packages/app/src/app/pages/Dashboard/Content/CreateNewSandbox/index.js

```diff
@@ -35,11 +35,16 @@ class CreateNewSandbox extends React.PureComponent {
     document.removeEventListener('keydown', this.keydownListener);
   }
 
-  createSandbox = template => {
-    this.setState({ forking: true }, () => {
+  createSandbox = (template, e) => {
+    const cmd = e.ctrlKey || e.metaKey;
+    const url = sandboxUrl({ id: template.shortid });
+    if (cmd === true) {
+      return window.open(url, '_blank');
```

Comment:
This will break creating sandboxes from a subdirectory, the sandbox will then not be forked/put in the directory. I think this one should be moved to where `history.push`is as well (which we do if the sandbox doesn't have a subdirectory).

The logic currently is:

If a sandbox is created without a subdirectory we just redirect you to the sandbox. If the sandbox is created from a subdirectory we actually create a new sandbox with the subdirectory properly set.